config/initializers/cors.rb

```diff
@@ -2,5 +2,6 @@
   allow do
     origins '*'
     resource '/api/v1/*', headers: :any, methods: :any
+    resource '/graphql/*', headers: :any, methods: :post
```

Comment:
Do not agree. On our concept in the v1 folder is the version of our api, then in the v2 folder there will be another version of api, for using the application on other protocols and on other principles, but the functionality in the same will remain, this means that api/graphql requires another version of api by the standards of graphql, we can not change the standard and will not.
 It's not logical to do v1/ graphql, v2/graphql etc